integration/v7/isolated/service_command_test.go

```diff
@@ -171,17 +171,16 @@ var _ = Describe("service command", func() {
 				})
 
 				It("displays the bound apps", func() {
-					//Delay to reduce flakiness
-					time.Sleep(10 * time.Second)
-					session := helpers.CF(serviceCommand, serviceInstanceName, "-v")
-					Eventually(session).Should(Exit(0))
-
-					Expect(session).To(SatisfyAll(
-						Say(`Showing bound apps:\n`),
-						Say(`name\s+binding name\s+status\s+message\n`),
-						Say(`%s\s+%s\s+create succeeded\s*\n`, appName1, bindingName1),
-						Say(`%s\s+%s\s+create succeeded\s*\n`, appName2, bindingName2),
-					))
+					Eventually(func(g Gomega) {
+						session := helpers.CF(serviceCommand, serviceInstanceName, "-v").Wait()
+						g.Expect(session).Should(Exit(0))
+						g.Expect(session).Should(SatisfyAll(
+							Say(`Showing bound apps:\n`),
+							Say(`name\s+binding name\s+status\s+message\n`),
+							ContainSubstring(`%s\s+%s\s+create succeeded\s*\n`, appName1, bindingName1),
+							ContainSubstring(`%s\s+%s\s+create succeeded\s*\n`, appName2, bindingName2),
+						))
```

Comment:
```suggestion
						))
						sessionContents := string(session.Out.Contents())
						Expect(strings.Contains(sessionContents, fmt.Sprintf(`%s   %s   create succeeded`, appName1, bindingName1))).To(BeTrue())
						Expect(strings.Contains(sessionContents, fmt.Sprintf(`%s   %s   create succeeded`, appName2, bindingName2))).To(BeTrue())
```

Reply:
I found the test can fail if app2 is listed first. If we get the session contents and search it for substrings the order doesn't matter.